build-system/server/new-server/transforms/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compileOnSave": true,
   "compilerOptions": {
-    "lib": ["esnext"],
+    "lib": ["esnext", "dom"],
```

Comment:
@rsimha please review, this fixes the issue

Reply:
This code is used by `gulp serve --new_server`. I'm not sure why it affects this PR. Are you starting a server somewhere? Could it be a red herring? (Maybe revert and see if it makes a difference?)

/cc @kristoferbaxter